sandbox/api/FHIR/R4/index.js

```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import response0 from './responses/empty-response.fhir';
```

Comment:
variable naming needs to mean something responseEmpty etc. Small one as it make sense what your importing just good to keep it consistent.

Reply:
Done